fury/tests/test_decorators.py

```diff
@@ -49,3 +49,29 @@ def f():
     del HAVE_AMODULE
     f.__doc__ = docstring
     npt.assert_raises(NameError, doctest_skip_parser, f)
+
+
+def test_warn_on_args_to_kwargs():
```

Comment:
I'm not sure if tests to check `from_version` and `until_version` are trivial. It'll be good if they are possible.

I think we could do something like `fury.__version__` to simulate the situation.